docs/Reference/Validation-and-Serialization.md

```diff
@@ -390,6 +390,11 @@ body, URL  parameters, headers, and query string. The default
 [ajv](https://ajv.js.org/) validation interface. Fastify uses it internally to
 speed the validation up.
 
+> Since `validatorCompiler` is loaded on the first schema validation it's possible
+> that its value is falsey. If you need to validate some schema at runtime
+> (in contexts not covered by Fastify by default)
+> take a look a [these helpers](https://fastify.dev/docs/latest/Reference/Request/#validateinputdata-schema--httpstatus-httpstatus)
```

Comment:
Why `>`?

Reply:
In order to mark that paragraph as a "note". I can format it as normal text if it's better

Reply:
We typically indicate that a note is a note:

https://github.com/fastify/fastify/blob/9a019624d4ca408ed58b00c5642a05d405b7d67c/docs/Reference/Server.md?plain=1#L1514-L1520

Reply:
Ok thanks